src/components/Atoms/Input/input.test.js

```diff
@@ -80,12 +80,12 @@ it('renders correctly', () => {
 
     <label
       className="c0"
-      htmlFor="Acessibility info go here"
+      htmlFor="undefined"
```

Comment:
Undefined? Why all the changes in the snaps?

Reply:
I just the same from pulling master, not sure what's going on

Reply:
I pulled the master

Reply:
`lazyload c5` to `c5 lazyload` and `C2` to `C1` are fine but `htmlFor="undefined"` is a bit worrying .. I will reinstall modules and run test in master before we merge those changes in all PRs.